README.md

```diff
@@ -0,0 +1,70 @@
+Hm-admin
+-----------
+
+
+## Description
+The main goal of this project is to provide a self-generated backoffice which take advantage of the context describe 
+  in Hypermedia API. This backoffice should be easy to override via simple configuration variables but must have a good 
+  abstraction to allow a deep override if the integrator need it.
+
+The backoffice should respect some rules:
+- Keep it simple: this project will not cover all the weirdo cases that your client will invent. It's a basis. Huge enough
+ to allow a lot of people to use it with small configuration, but a small basis to contains only required features. 
+ (In a second time, the app may be able to allow plugins to increase the features but first things first).
+- Keep it clean: this project, by default, respect some auto-discipline (coding guidelines, high code coverage ratio, ...).
+  As this project becomes your project, it's not mandatory to follow them but if you want, you can find them 
+  right [here][guidelines].
+- Keep it User-friendly: this project follows the [Material design recommandations][material-design] to provide a full 
+ responsive and nice experience for the final user. It's not a matter about feeding the troll on the Apple/Android
+ war. Those recommendations made their proofs so we use them.
+
+> As the full stack of typescript + compilation + tests + ... is quite pain in the ass to install, this project is based 
+  on [angular-cli][angular-cli].
+
+## Let's start
+### Requirements
+- Git
+- NPM (we don't use leftpad so everything should be fine ;))
+- An Hypermedia API (Have you ever heard of [API Platform][api-platform]?)
+
+### Installation
+
+First, clone the project
+```
+git clone git@github.com:coopTilleuls/hm-admin.git name-of-your-project 
+cd name-of-your-project
+```
+
+> Note from Captain Obvious: Don't forget to replace `name-of-your-project` by your project's name.
+
+Then you can install dependencies and start your project
+```
+npm install -g angular-cli
+npm start
+```
+
+And that's it...
+
+### Configuration
```

Comment:
Yep, I just take the hydra admin readme and make some search & replace. You can make a release without this feature, it will come in the next one